src/components/ColorPalette/index.js

```diff
@@ -175,33 +41,27 @@ const ColorPalette = () => {
                 </div>
             </div>
             <div className="color-palette-inner-container">
-                <div>GREEN</div>
+                <div className="color-name">GREEN</div>
                 <div className="colors-container">
-                    {palette.map((greenPalette) =>
+                    {PALETTE.map((greenPalette) =>
```

Comment:
```suggestion
                    {PALETTES.map((PALETTE) =>
```
atau bisa juga di desctructure
```suggestion
                    {PALETTES.map(({GREEN}) =>
```
HMM namanya kok kayaknya ga sesuai ya

Reply:
sudah aku ganti pakai looping ya ini ko Randi